index.html

```diff
@@ -0,0 +1,359 @@
+            <img src="./image/ice-cream-mobil.png" alt="ice-cream">
+          </picture>
+          <h1>
+            ice cream <br> made with<br><span> passion <span>
+          </h1>
+          <ul class="list">
+            <li class="item"><button>Products</button></li>
+            <li class="item"><button>How it’s made?</button></li>
+          </ul>
+          <div class="tablet-style">
+            <picture  class="milk">
+              <source srcset="./image/milk1200.png" media="(min-width:1120px)">
+              <source srcset="./image/milk.png" media="(min-width:768px)">        
+              <img src="./image/milk.png"  alt="milk">
+            </picture>
+            <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et</p>
+            <button class="arrow">
+              <picture>
+                <source srcset="./image/arrow1200.png" media="(min-width:1120px)">
+                <source srcset="./image/arrow-right.png" media="(min-width:768px)">     
+                <img src="./image/arrow-right.png" alt="раскрыть текст">
+              </picture>         
+            </button>
+            <picture class="girl">
+              <source srcset="./image/girlDesctop.png" media="(min-width:1120px)">
+              <source srcset="./image/girl.png 1x, ./image/girl@2x.png 2x" media="(min-width:768px)">
+              <img class="girl-ice" src="./image/girl.png" alt="girl">
+            </picture>
+            <ul class="count">
+              <li class="item">
+                <svg class="cafe" width="43" height="44">
+                  <use href="./image/svg-header.svg#icon-Group-22"></use>
+                </svg>
+              </li>
+              <li class="item">
+                <svg class="food" width="51" height="43">
+                  <use href="./image/svg-header.svg#icon-Group-23"></use>
+                </svg>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section>
+        <div class="container">
+          <p class="product-description">100% natural</p>
+          <h2 class="product-title">products</h2>
+          <ul class="products">
+            <li class="item">
+              <h3 class="item-name">ice cream</h3>
+              <ul class="menu-cicle">
+                <li class="cicle"><img src="./image/EllipseLittle.png"></li>
+                <li class="cicle"><img src="./image/EllipseLittle.png"></li>
+                <li class="cicle"><img src="./image/EllipseLittle.png"></li>
+              </ul>
+              <p class="item-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et. dolore magna
+              aliqua facilisis.</p>
+              <button class="product-arrow">
+                <img src="./image/arrow40.png" alt="стрелка">
```

Comment:
it is better to have such attributes as width and height for the tag IMG

Reply:
add